src/flows/recommendation_api/legacy_topics_flow.py

```diff
@@ -0,0 +1,70 @@
+from typing import List
+from prefect import Flow, Parameter, unmapped, task
+from prefect.executors import LocalDaskExecutor
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery, OutputType
+from common_tasks.corpus_candidate_set import (
+    validate_candidate_items,
+)
+from utils import config
+from utils.flow import get_flow_name, get_interval_schedule
+from api_clients.sqs import put_results, RecommendationCandidate, NewTabFeedID
+
+FLOW_NAME = get_flow_name(__file__)
+
+# Export approved candidate items by language and recency
+EXPORT_CORPUS_ITEMS_SQL = """
+SELECT 
+    a.resolved_id as "ID", 
+    c.top_domain_name as "PUBLISHER"
+FROM "ANALYTICS"."DBT"."APPROVED_CORPUS_ITEMS" as a
+JOIN "ANALYTICS"."DBT".content as c ON c.content_id = a.content_id
+WHERE REVIEWED_CORPUS_ITEM_UPDATED_AT >= DATEADD('day', -90, current_timestamp())
+AND a.TOPIC = %(CORPUS_TOPIC_ID)s
+AND a.LANGUAGE = 'EN'
+"""
+
+GET_TOPICS_SQL = """
+select 
```

Comment:
Im finding understanding what this query very difficult. Perhaps the table names should be more clear and we should add `corpus_topic_id` to the other table so we don't have to join.

Reply:
Yeah, the name “topic map guids” is not very descriptive, unless you know there is a file in metaflow named like this. G, maybe you can create a static table with only the data that we need, and choose descriptive names without being ties to how things were named in Metaflow?

Reply:
Yes. We can create a new static table in Snowflake dbt with better naming and just the 2 needed columns. We also need to include the missing metadata for the `CORONAVIRUS` topic in the table